connector/src/main/resources/config.yml

```diff
@@ -25,6 +25,10 @@ bedrock:
   # Whether to enable PROXY protocol or not for clients. You DO NOT WANT this feature unless you run UDP reverse proxy
   # in front of your Geyser instance.
   enable-proxy-protocol: false
+  # A list of allowed PROXY protocol speaking proxy IP addresses/subnets. Only effective when "enable-proxy-protocol" is enabled, and
+  # should really only be used when you are not able to use a proper firewall (usually true with shared hosting providers etc.).
+  # Keeping this list empty means there is no IP address whitelist.
+  proxy-protocol-whitelisted-ips: []
```

Comment:
I would comment this out and illustrate how IPs should properly be formatted in.